packages/client/hmi-client/src/services/charts.ts

```diff
@@ -32,6 +32,185 @@ export interface HistogramChartOptions extends BaseChartOptions {
 	variables: { field: string; label?: string; width: number; color: string }[];
 }
 
+export interface ErrorChartOptions extends Omit<BaseChartOptions, 'height' | 'yAxisTitle' | 'legend'> {
+	maxBins?: number;
+	height?: number;
+	areaChartHeight?: number;
+	boxPlotHeight?: number;
+	variables: { field: string; label?: string }[];
+}
+
+export const createErrorChart = (dataset: Record<string, any>[], options: ErrorChartOptions) => {
+	const axisColor = '#EEE';
+	const labelColor = '#667085';
+	const labelFontWeight = 'normal';
+	const globalFont = 'Figtree';
```

Comment:
Not for this PR, but we should make those constants for ALL the charts on the application.

Reply:
That makes sense, we can probably need to revisit this and do some clean ups